src/svg/Painter.ts

```diff
@@ -143,7 +143,7 @@ class SVGPainter implements PainterBase {
 
         const children: SVGVNode[] = [];
 
-        const bgVNode = this._bgVNode = createBackgroundVNode(width, height, this._backgroundColor, scope);
+        const bgVNode = this._bgVNode = createBackgroundVNode(this._id, width, height, this._backgroundColor, scope);
```

Comment:
I always feel this id is unnecessary as I can't see its useful points and the user can't set it in some way. Perhaps we can remove it directly.